ebos/eclproblem.hh

```diff
@@ -972,6 +972,7 @@ public:
      */
     void beginEpisode()
     {
+        OPM_TIMEBLOCK(beginEpisode);
```

Comment:
I think this should indicate that it is the beginEpisode of EclProblem, since there are many classes that have a beginEpisode method (right?). I do not know if the Tracy macros allow arbitrary names, but `EclProblem::beginEpisode` would be my preferred version, `EclProblem_beginEpisode` if that is not allowed.

The same applies to begin/end Iteration, Timestep etc.

I see you already used a similar construction below: `eclProblemSource`, I would prefer either `EclProblem::source` or `EclProblem_source` to have a direct match between names.

Reply:
:: is not allowed. The other is ok. There is beginEpisode problems. In the original design it should only be in the problem, but our models (i.e. aquifers and wells are kind of a mix).